pkg/cmd/tracerunner.go

```diff
@@ -13,15 +13,51 @@ import (
 	"syscall"
 
 	"github.com/fntlnz/mountinfo"
+	"github.com/iovisor/kubectl-trace/pkg/tracejob"
 	"github.com/spf13/cobra"
 )
 
+const (
+	bpftrace = "bpftrace"
+	bcc      = "bcc"
+)
+
+var (
+	bpfTraceBinaryPath = "/bin/bpftrace"
```

Comment:
is this where we install it to in the container? I think /usr/bin/bpftrace is more common, so we should standardize if that is the case.

Reply:
No you are correct it is actually installed at `/usr/bin/bpftrace` I copied it without checking from the flag below.